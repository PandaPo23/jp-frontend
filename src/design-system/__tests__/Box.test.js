import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Box from '../Box';

describe('Box component', () => {
  it('should render as box', () => {
    const wrapper = mount(<Box />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
