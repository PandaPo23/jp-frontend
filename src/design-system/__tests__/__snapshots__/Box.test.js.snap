// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box component should render as box 1`] = `
<Box>
  <div
    className="css-sb275m"
  />
</Box>
`;
